spec/requests/apartments_spec.rb

```diff
@@ -2,6 +2,13 @@
 
 RSpec.describe "Apartments", type: :request do
   describe "GET /index" do
-    pending "add some examples (or delete) #{__FILE__}"
+    it "get a list of apartments" do
+        user = User.where(email: 'test@example.com').first_or_create(password:'12345678', password_confirmation:'12345678')
+        user.apartments.create street: "1234 Street Lane", city: 'San Diego', state: 'CA', manager: 'Elyse', email: 'elyse_manager@email.com', price: '3000', bedrooms: 1, bathrooms: 1, pets: '0', image: 'image', user_id: 1
+        get '/apartments'
```

Comment:
This may be throwing your no method error.  Let's remove this spec and the controller method for index for this PR.  Keep your model specs though as these look solid.